### PR TITLE
Tappx: rewrite schema in case of protocol in host param

### PR DESCRIPTION
**Pull request by RodionOrets**

### CODE REVIEW

File: src/main/java/org/prebid/server/bidder/tappx/TappxBidder.java [hunk 1]
```diff
@@ -130,6 +130,12 @@ private String buildUrl(String host, String endpoint, String tappxkey, Integer t
         }
     }
 
+    private String resolveHost(String host) {
+        return StringUtils.containsIgnoreCase(host, "http://") || StringUtils.containsIgnoreCase(host, "https://")
```
Comment (RodionOrets) [inline src/main/java/org/prebid/server/bidder/tappx/TappxBidder.java:134]: Decided to add check exactly for `http://` or `https://`. Checking only for `.contains("http")` could lead to problems with urls like this one: `httphost.com/...`. Word `http` could be the actual part of host. But if it's a mistake of a PBJ caller - it's a PBJ caller problem :)

File: src/test/java/org/prebid/server/bidder/tappx/TappxBidderTest.java [hunk 2]
```diff
@@ -191,6 +191,29 @@ public void makeHttpRequestShouldBuildCorrectUriWithPathInHostParameterAndSlashB
                 });
     }
 
+    @Test
+    public void makeHttpRequestShouldBuildCorrectUriWithSchemeInHostParam() {
+        // given
+        final BidRequest bidRequest = BidRequest.builder()
+                .imp(singletonList(Imp.builder()
+                        .ext(mapper.valueToTree(ExtPrebid.of(null,
+                                ExtImpTappx.of("https://host.com/rtb/v2", "tappxkey", "/endpoint", BigDecimal.ONE))))
```
Comment (RodionOrets) [inline src/test/java/org/prebid/server/bidder/tappx/TappxBidderTest.java:200]: Included all of possible corner cases to this request params

File: src/main/java/org/prebid/server/bidder/tappx/TappxBidder.java (hunk 1 above)
Comment (rpanchyk) [reply src/main/java/org/prebid/server/bidder/tappx/TappxBidder.java:134]: We even can use `StringUtils.startsWithIgnoreCase()` to ensure the beginning of the url.
Or even: `StringUtils.startsWithAny(host.toLowerCase(), "http://", "https://")`.
Up to you, just a syntax sugar.

**Review by rpanchyk — CHANGES_REQUESTED** [review]

**Review by rpanchyk — APPROVED** [review]
